sdk/storage/blobs/inc/az_storage_blobs.h

```diff
@@ -42,6 +56,19 @@ typedef struct
   } _internal;
 } az_storage_blobs_blob_client;
 
+
+/**
+ * @brief Init a client with default options
+ *
+ * @param client blob_client to initialize
+ * @param url a url to a blob storage account
+ * @param credential credential object for authentication
+ *         #AZ_CREDENTIAL_ANONYMOUS should be used for SAS
+ * @param options client options
```

Comment:
Taking inspiration from other places where we init an object passing in `options` pointer.

Note, you'll likely have to format this file with Ctrl+K/Ctrl+D after you accept this suggestion to get line wrapping since it will go over 100 characters. 
```suggestion
 * @param options  A reference to an #az_storage_blobs_blob_client_options structure which defines custom behavior of the client.
```